### PR TITLE
updated title of readme

### PR DESCRIPTION
**Pull request by Ynoctis**
changed title format

readme runs correctly

### CODE REVIEW

**Review by amazingbrian — CHANGES_REQUESTED** [review]
absolutely trash

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# silverash-mythicplus-bot
+# Silverash MythicPlus Bot
 M+ bot for Silverash
```
Comment (mus1cholic) [inline README.md:2]: nit: change the description to be a little bit more detailed as well

**Review by mus1cholic — CHANGES_REQUESTED** [review]

**Review by mus1cholic — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (amazingbrian) [reply README.md:2]:
```suggestion
M+ bot for SilverASS
```

**Review by amazingbrian — APPROVED** [review]
```suggestion
M+ bot for SilverASS
```

**Pull request merged**